swift/swift.conf

```diff
@@ -2,3 +2,6 @@
 # random unique strings that can never change (DO NOT LOSE)
 swift_hash_path_prefix = veDDnRGXVW7rXqc3
 swift_hash_path_suffix = 0rBOk3EM17yId7Ke
+
+[swift-constraints]
+max_meta_value_length = 2048
```

Comment:
If the goal is to make Swift constraints match S3, there are a few more relevant settings: https://github.com/openstack/swift/blob/master/swift/common/constraints.py

Specifically, I think you want to set max_meta_overall_size to 2048 and max_meta_header_size to 2048. It looks like S3 doesn't have a hard constraint on the length of the metadata key either (at least I can't find it here: https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingMetadata.html#object-metadata), so maybe max_meta_name_length should also be 2048. Can you validate the various lengths against AWS S3? I'm also unclear if there is a maximum number of x-amz-meta- headers allowed (which Swift currently sets to 90).